ci/scripts/python_util.sh

```diff
@@ -21,6 +21,13 @@ set -ex
 
 COMPONENTS="adbc_driver_manager adbc_driver_flightsql adbc_driver_postgresql adbc_driver_sqlite"
 
+if [[ $(uname) = "Darwin" ]] && [[ "${VCPKG_ARCH}" = "arm64" ]]; then
+    echo "Disabling adbc_driver_snowflake"
+    echo "(snowflake uses arrow v11 which can't be compiled on non-x64 platforms without noasm)"
```

Comment:
should we add the `noasm` flag for those platforms in order to still allow it to get built?

Reply:
I can try that but want to at least get CI green once here